src/jvm/test/resources/expected_logs/run_concurrent_test/main_thread_blocked_trace_debugger_jdk13.txt

```diff
@@ -1 +1,66 @@
 = Concurrent test has hung =
+
+The following interleaving leads to the error:
+| -------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                      Thread 1                                                                      |
+| -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| $this_runCatching.block() at BaseRunConcurrentRepresentationTest.testRunWithModelChecker$lambda$1$lambda$0(RunConcurrentRepresentationTests.kt:47) |
+|   block() at MainThreadBlockedRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:96)                                        |
+|     q.take() at MainThreadBlockedRunConcurrentRepresentationTest.block(RunConcurrentRepresentationTests.kt:101)                                    |
+|       lock.lockInterruptibly() at ArrayBlockingQueue.take(ArrayBlockingQueue.java:414)                                                             |
+|       count ➜ 0 at ArrayBlockingQueue.take(ArrayBlockingQueue.java:416)                                                                            |
+|       ConditionObject#1.await() at ArrayBlockingQueue.take(ArrayBlockingQueue.java:417)                                                            |
+|         addConditionWaiter(): Node#1 at AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:2077)                     |
```

Comment:
Should AQS detail be in a silent section?

Reply:
No I think it shouldn't. I haven't updated this file yet, on it.